Makefile

```diff
@@ -847,6 +847,10 @@ else
 	 go run update_kubernetes_version.go)
 endif
 
+.PHONY: stress
+stress:
```

Comment:
add ## helper for what it means so it shows up in "make help"

Reply:
done